integrations/observability/nginx/getting-started/nginx-getting-started-1.0.0.json

```diff
@@ -0,0 +1,58 @@
+    },
+    {
+      "name": "Fluent-Bit Setup",
+      "label": "Agent Set-Up",
+      "info": "<info URL>",
+      "description": "Setup Fluent-Bit conf file including logs parsing and OpenSearch access",
+      "content": "[SERVICE]\n    Parsers_File parsers.conf\n\n[INPUT]\n    Name forward\n    Port 24224\n\n[FILTER]\n    Name parser\n    Match nginx.access\n    Key_Name log\n    Parser nginx\n\n[FILTER]\n    Name parser\n    Match apache.access\n    Key_Name log\n    Parser apache\n\n[Filter]\n    Name    lua\n    Match   *\n    Script  otel-converter.lua\n    call    convert_to_otel\n\n[OUTPUT]\n    Name  opensearch\n    Match nginx.access\n    Host  opensearch-node1\n    Port  9200\n    Index ss4o_logs-nginx-prod\n    Suppress_Type_Name On\n    tls             On\n    tls.verify      Off\n    HTTP_User       admin\n    HTTP_Passwd     my_%New%_passW0rd!@#\n\n[OUTPUT]\n    Name  opensearch\n    Match apache.access\n    Host  opensearch-node1\n    Port  9200\n    Index ss4o_logs-apache-prod\n    Suppress_Type_Name On\n    tls             On\n    tls.verify      Off\n    HTTP_User       admin\n    HTTP_Passwd     my_%New%_passW0rd!@#\n\n[OUTPUT]\n    Name stdout\n    Match nginx.access"
+    }
+  ],
+  "statics": {
+    "logo": [{
+      "annotation": "Fluent-Bit Logo",
+      "path": "logo.svg"
+    },
+      {
+        "annotation": "Nginx Logo",
+        "path": "logo.svg"
+      }]
+  },
+  "relatedAssets": [
+    {
+      "name": "nginx-integration",
+      "type": "integration",
+      "version": "1.0.0",
+      "displayName": "Nginx-Integration",
+      "catalog": "Observability",
+      "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/nginx/info"
+    }
+  ],
+  "liveDemo": {
+    "command": "docker-compose up -d",
+    "path": "<docker-compose-url>"
```

Comment:
Another placeholder

Reply:
thanks - moved into draft util all the comments / questions are resolved